images/build-root/Dockerfile.custom

```diff
@@ -11,3 +12,7 @@ RUN /usr/local/bin/operator-sdk version
 RUN curl -s -L https://github.com/kubernetes-sigs/kustomize/releases/download/kustomize%2F${KUSTOMIZE_VERSION}/kustomize_${KUSTOMIZE_VERSION}_linux_amd64.tar.gz | tar xvzf - -C /usr/local/bin kustomize
 RUN chmod +x /usr/local/bin/kustomize
 RUN /usr/local/bin/kustomize version
+RUN curl -s -L "https://github.com/kudobuilder/kuttl/releases/download/${KUTTL_VERSION}/kubectl-kuttl_${KUTTL_VERSION#v}_linux_x86_64" -o kubectl-kuttl
```

Comment:
My concern with this change is that we are adding a third variable to build_root image and also adding kuttl binary to all other jobs (pre-commit, units, linters, ...)
What about adding it on ci-op config files?
if we add a new image to all ci-config files, using ARG to set its version, It should be easy to update all KUTTLE_VERSION in a single patch on openshift/release too.

Reply:
If we add it here, I think that will be harder to validate if a new KUTTL_VERSION works/breaks repos.
We will need to bump it here, and after merging the patch, it will generate a new root_image. This new image will affect all repos that use it as built_root image.
If that doesn't work for a single repo, we will need to rollback the image.

Another way would be to create a new image with a new version of KUTTL bins, and then, go into those repos and update their root_image (which will need to be done one by one).

In the end, I don't see as the best option.